src/BrefKernel.php

```diff
@@ -122,6 +123,24 @@ protected function prepareCacheDir(string $readOnlyDir, string $writeDir): void
         ));
     }
 
+    /**
+     * Even though applications should never write into it on Lambda, there are parts of Symfony
+     * (like "about" CLI command) that expect the log dir exists, so we have to make sure of it.
+     *
+     * @see https://github.com/brefphp/symfony-bridge/issues/42
+     */
+    protected function ensureLogDir(string $writeLogDir): void
+    {
+        if (! $this->isLambda() || is_dir($writeLogDir)) {
```

Comment:
```suggestion
        if (is_dir($writeLogDir) || ! $this->isLambda()) {
```

I _think_ `is_dir()` is faster to run, so it might be better to call this one first (that way if the directory exists, it will skip the next test).

Reply:
I don't know. But if, then the same is probably true for the condition in `prepareCacheDir()`…

Reply:
Really not sure about performance impact here. But we're not talking about compute heavy things in either case, `isLambda` is just a string comparison with a environment variable.

Reply:
And just for fun, I made a very naive benchmark and apparently `is_dir` is slower. Not to be taken as a scientific measure though, but `isLambda` would be around a 100 times faster than `is_dir`.

See https://3v4l.org/d4ILY#v8.0.8

Reply:
Good to know thanks!